src/main/java/duke/Deadline.java

```diff
@@ -0,0 +1,32 @@
+package duke;
+
+import java.time.LocalDate;
+import java.time.format.DateTimeFormatter;
+
+public class Deadline extends Task {
+
+    protected LocalDate by;
+
+    public Deadline(String description, String by) {
+        super(description);
+        this.by = LocalDate.parse(by);
+    }
+
+    /**
+     * Returns the string representation of a Deadline instance in MMM d yyyy format.
+     * @return the desired string representation of a Deadline instance.
```

Comment:
Perhaps you should have an empty line between description and parameter section?

Reply:
Thanks for pointing that out! I wasn't aware of this coding standard